src/main/scala/esmeta/spec/util/Parser.scala

```diff
@@ -88,7 +88,7 @@ trait Parsers extends LangParsers {
 
   /** terminals */
   lazy val term: Parser[Terminal] = {
-    "`[^`]+`|```".r ^^ {
+    "`[^`]+`|`[`]+`".r ^^ {
       case str =>
         Terminal(str.substring(1, str.length - 1))
```

Comment:
Shouldn't `1` be replaced with the length of the delimiter here?

Reply:
When would the length of the delimiter not be 1?

Reply:
I'm unsure what 4 or more backticks in a row is supposed to mean. It doesn't appear to be legal. So I don't know.

Reply:
I'm reading [the code for grammarkdown](https://github.com/rbuckton/grammarkdown/blob/2ecf056a1b774733afd73113533dfad9f7ad44c8/src/scanner.ts#L833-L860) and it doesn't seem it would support 4 backticks in a row, though I could be misreading. Does it have something to do with tc39/ecma262#2418 using [`&grave;`](https://github.com/tc39/ecma262/pull/2418/files#diff-181371b08d71216599b0acccbaabd03c306da6de142ea6275c2135810999805aR35056) and the entities being processed by esmeta but not grammarkdown?

Reply:
ESMeta unescapes the HTML entities before parsing the `spec.html`. Thus, the text `` `&grave;&grave;` `` is unescaped as: `` ```` ``, and ESMeta parses the four backticks (`` ```` ``) as a literal terminal consists of two backticks (``` `` ```) because the first and last backtick is used to represent literal terminals. I supported this feature because https://github.com/tc39/ecma262/pull/2418 defines the literal terminal consisting of two backticks. Is there any problem with this extension?

Reply:
Okay, that should be fine then. I didn't know about the pre-processing of HTML entities by esmeta.